src/network/networkd-address.c

```diff
@@ -797,6 +797,10 @@ int config_parse_label(
 
         _cleanup_address_free_ Address *n = NULL;
         Network *network = userdata;
+        char ifname[IF_NAMESIZE] = "";
```

Comment:
No need to initialize this.

`k` should be scoped under the `if` where it is used.

`p` should be `const` too.